content/blog/Gone-phishing

```diff
@@ -0,0 +1,149 @@
+
+# Gophish-evilginx2 trip
+
+Disclaimer
+
+*All information in this blog post is only for educational purposes or legitimate penetration testing assignments with written permissions.*
+
+## Introduction
+
+Phishing is categorized as social engineering and is used to steal login credentials. If done right phishing can even bypass 2FA and therefore can be very dangerous. Today's trip will be all about Email phishing and spear phishing with the right tools and the best bait!
+
+### Risky business
+
+The impact of successful phishing attacks on a business can be devastating. From loss of money and intellectual property to damage of the company's reputation and disruption of day to day work.
+
+### What to bring on your phishing trip
+
+* Domain name used for phishing.
+* Internet facing virtual private server
+* A MITM proxy which can capture credentials (evilginx2)
+* Phishing framework for email templates and phishing campaigns.(GoPhish)
+* SMPT server (sendgrid)  
```

Comment:
typo, SMTP @Doomed5G . Sendgrid doesn't matter actually, but we should mention we are going to use their free tier just because it's free.